src/gwt/.idea/misc.xml

```diff
@@ -33,6 +30,9 @@
     </autodetection-disabled>
   </component>
   <component name="IdProvider" IDEtalkID="AC9390DF9D3D1C907504F5601CCC6695" />
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
+  </component>
```

Comment:
I'm realizing now that we may not actually support ES6 entirely outside of the desktop environment. Will confirm with ya'll tomorrow if I need to ES5ify this stuff.

Reply:
We do need to support ES5, unfortunately; there are a few environments where IE11 support is required. However the IDE only has to be functional/usable in these environments; in my view the work in this PR generally falls under the category of performance optimization. So long as you still get corrections in IE11 it's okay if you have to wait a little longer.